api/sensor.py

```diff
@@ -127,8 +131,8 @@ async def delete_sensor(
 async def get_sensors(
     session: session_dependency,
     user: viewer_dependency,
-    thing_id: int = None,  # Optional filter for thing_id
-    observed_property: str = None,  # Optional filter for observed_property
+    thing_id: int = None,  # Optional filter for thing_id. Filter by the Thing where equipment is deployed
+    parameter_id: int = None,  # Filter by the parameter the sensor/equipment measures
```

Comment:
Since these query parameters are now present in this endpoint we should make tests to ensure they work as expected and to safeguard future development.